unit-tests/live/aus/test-D415.py

```diff
@@ -0,0 +1,56 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test:device D415

```

Comment:
Do we have D415 on LibCI?
I am not sure..
Please find out and align to a connected device if D415 is not.